lts.json

```diff
@@ -14,5 +14,5 @@
       "ember-d3": "0.2.0"
```

Comment:
Package is to install only a single package.
Group is to install a group of packages that must be install together (compatibility between version or dependencies, etc.)

The empty packages and groups are only for testing purposes. I will remove them when we will start using this tool.